scripts/osfstorage/files_audit.py

```diff
@@ -127,11 +134,19 @@ def main(nworkers, worker_id, dry_run):
     progress_bar.finish()
 
 
-if __name__ == '__main__':
-    import sys
-    nworkers = int(sys.argv[1])
-    worker_id = int(sys.argv[2])
-    dry_run = 'dry' in sys.argv
+@celery_app.task(name='scripts.osfstorage.files_audit')
+def run_mains(num_of_workers, dry_run):
+    try:
+        thread.start_new_thread(run_main, (num_of_workers, 0, dry_run,))
```

Comment:
It appears this task might be better spawning additional Celery tasks, so you won't need to manage the threads.